### ISSUE TITLE
Only the statements that execute successfully to be added to a history

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#1947]
Original [issue 1947](https://code.google.com/p/sqlyog/issues/detail?id=1947) created by webyog on 2013-06-17T06:44:12.000Z:

User requested &quot; I write a lot of sql statements against different databases and for different purposes. Keying them again and again gets old. I would like to have the statements that execute successfully to be added to a history that supports querying by database, table(s), column(s), statement type, comments, and so forth. When selected it would be nice to have the statement be setup for editing and execution. Statements should be remembered only once and variations only due to constants should be ignored.&quot;

Refer: http://forums.webyog.com/index.php?showtopic=7168
